include/ddc/discrete_space.hpp

```diff
@@ -238,15 +251,11 @@ KOKKOS_FUNCTION detail::ddim_impl_t<DDim, MemorySpace> const& discrete_space()
     }
 }
 
-template <class DDim>
-bool is_discrete_space_initialized() noexcept
-{
-    return detail::g_discrete_space_dual<DDim>.has_value();
-}
-
 template <class DDim>
 detail::ddim_impl_t<DDim, Kokkos::HostSpace> const& host_discrete_space()
 {
+    // Calling this function requires to call first `ddc::init_discrete_space<DDim>(...);`
```

Comment:
Grammar improvement (same above)
```suggestion
    // This function requires that `ddc::init_discrete_space<DDim>(...);` be called first
```